config/ember-try.js

```diff
@@ -1,24 +1,25 @@
 'use strict';
 
 const getChannelURL = require('ember-source-channel-url');
+const { embroiderSafe, embroiderOptimized } = require('@embroider/test-setup');
 
-module.exports = async function() {
+module.exports = async function () {
   return {
     useYarn: true,
     scenarios: [
       {
-        name: 'ember-lts-3.13',
```

Comment:
If we're dropping support for these older versions, we'll have to mention it in the release notes and release it as a new major version

Reply:
Meant to add those back after messing with circleci's config, it *should* still support them